website/README.md

```diff
@@ -24,6 +24,27 @@ If you are not familiar with Jekyll, be sure to check out its
 [documentation][jekyll-docs]. It is recommended to follow the provided
 step-by-step tutorial.
 
+###### Switch Ruby versions
+
+The required Ruby version is set in `.ruby-version`. To switch, you can use
+[rvm][rvm] to manage your Ruby version.
+
+###### Resolve Bundler issues
```

Comment:
@japborst I changed the grammar of these headers on purpose: the topic is "switch&ZeroWidthSpace;_ing_ Ruby versions" and "resolve&ZeroWidthSpace;_ing_ Bundler issues". This also matches the format used for the top-level README. Any particular reason this was reverted?

Reply:
Reverted to be in line with the rest.

Reply:
The other headers do not use the imperative form either?